src/modules/expenses/expense.repository.ts

```diff
@@ -0,0 +1,50 @@
+import type { TInsertExpenseSchema } from "@/db/schemas/expense.model";
+
+import { db } from "@/db/adapter";
+import expenseModel from "@/db/schemas/expense.model";
+
+interface TExpensePayloadUser {
+  amount: TInsertExpenseSchema["amount"];
+  currency: TInsertExpenseSchema["currency"];
+  splitType: TInsertExpenseSchema["splitType"];
+  description?: TInsertExpenseSchema["description"];
+}
+
+interface TExpensePayloadAdmin {
```

Comment:
use uppercase I prefix for interfaces and not T, T is for types.

Reply:
Now using `type` so corrected.